Dockerfile

```diff
@@ -6,11 +6,16 @@ COPY Gemfile.lock /usr/src/app/
 
 RUN apk --update add ruby ruby-dev ruby-bundler build-base && \
     bundle install -j 4 && \
-    apk del build-base && rm -fr /usr/share/ri
+    apk del build-base && rm -fr /usr/share/ri && \
+    apk --update add git
```

Comment:
Is there a specific reason to do this here rather than adding `git` to the list above?